lock-impl/src/main/java/com/palantir/lock/impl/ThreadPooledLockService.java

```diff
@@ -15,31 +15,31 @@
  */
 package com.palantir.lock.impl;
 
+import java.io.IOException;
 import java.util.Set;
 import java.util.concurrent.Semaphore;
 
 import javax.annotation.Nullable;
 
+import com.palantir.lock.CloseableRemoteLockService;
 import com.palantir.lock.ForwardingLockService;
 import com.palantir.lock.HeldLocksToken;
 import com.palantir.lock.LockClient;
 import com.palantir.lock.LockRefreshToken;
 import com.palantir.lock.LockRequest;
 import com.palantir.lock.LockResponse;
 import com.palantir.lock.LockService;
```

Comment:
Some of these imports aren't used anymore which makes checkstyle unhappy. Might be worth configuring IntelliJ to optimize imports on the fly